library/core/src/ptr/mod.rs

```diff
@@ -2552,8 +2552,12 @@ mod verify {
     }
 
     #[kani::proof_for_contract(align_offset)]
-    fn check_align_offset_17() {
-        let p = kani::any::<usize>() as *const [char; 17];
+    // The purpose of this harness is to test align_offset with a pointee type whose stride is not a power of two
+    // Keeping the size smaller (5) and changing the solver to kissat improves verification time
+    // c.f. https://github.com/model-checking/verify-rust-std/pull/89
+    #[kani::solver(kissat)]
+    fn check_align_offset_5() {
```

Comment:
Can I please ask for the following additional data points to be provided?
1. You are making two changes at once. Can you instead make each change in isolation and provide the runtimes for those?
2. I am a bit surprised there is a big difference between Kani's default of cadical vs kissat. The above request will already help understand a bit better how much of a difference there really is. As we are playing with solvers, can you please also try minisat instead?

Reply:
| Stride | Solver | Runtime (sec) |
|-------|--------|-------------|
| 5 | cadical | 205.23
| 5| kissat |  58.33 |
| 5 | minisat | 528.62 |
| 17 | cadical | 1126.87 |
| 17 | kissat | 495.80 |
| 17| minisat | cut off after 2.5 hours |

Reply:
I still have minisat running in the background, so I'll come back and update this if it finishes by the end of the day.

Reply:
Decided to cut it off after 4hrs -- this is enough to get the point that minisat doesn't do well for this harness.